#include <iostream>

using namespace std;

void err_101()
{
    cout<<"101: The desired room was not found"<<endl;   
}
void err_102()
{
    cout<<"102: Your Reservation was not found"<<endl;   
}
void err_104()
{
    cout<<"104: Successfully added."<<endl;   
}
void err_105()
{
    cout<<"105: Successfully modified."<<endl;   
}
void err_106()
{
    cout<<"106: Successfully deleted."<<endl;   
}
void err_108()
{
    cout<<"108: Your account balance is not enough"<<endl;   
}
void err_109()
{
    cout<<"109: The room capacity is full"<<endl;   
}
void err_110()
{
    cout<<"110: Successfully done."<<endl;   
}
void err_111()
{
    cout<<"111: This room already exists"<<endl;   
}
void err_201()
{
    cout<<"201: User logged out successfully."<<endl;   
}
void err_230()
{
    cout<<"230: User logged in."<<endl;   
}
void err_231()
{
    cout<<"231: User successfully signed up."<<endl;   
}
void err_311()
{
    cout<<"311: User Signed up. Enter your password, purse, phone and address."<<endl;   
}
void err_312()
{
    cout<<"312: Information was changed successfully."<<endl;   
}
void err_401()
{
    cout<<"401: Invalid value!"<<endl;   
}
void err_403()
{
    cout<<"403: Access denied!"<<endl;   
}
void err_412()
{
    cout<<"412: Invalid capacity value!"<<endl;   
}
void err_413()
{
    cout<<"413: successfully Leaving."<<endl;   
}
void err_430()
{
    cout<<"430: Invalid username or password."<<endl;   
}
void err_451()
{
    cout<<"451: User already existed!"<<endl;   
}
void err_503()
{
    cout<<"503: Bad sequence of commands"<<endl;   
}